packages/commonwealth/client/scripts/views/components/component_kit/cw_sidebar_menu.tsx

```diff
@@ -2,18 +2,17 @@ import React, { useState } from 'react';
 
 import 'components/component_kit/cw_sidebar_menu.scss';
 
+import { featureFlags } from 'helpers/feature-flags';
+import { navigateToCommunity, useCommonNavigate } from 'navigation/helpers';
 import app from 'state';
-import AddressInfo from '../../../models/AddressInfo';
+import useSidebarStore from 'state/ui/sidebar';
```

Comment:
I can see some resorting in imports in this PR. Is this some automation or you do it manually?

Reply:
`Command + Shift + O` in vscode 😄, orders it automatically. Please lmk if i should revert.

Reply:
Actually its using this extension by default https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint&ssr=false 🤔.

I think we can set up a lint config that works cross IDEs. (not sure if its just my IDE alone that's messing up 😅)

Reply:
[Done](https://github.com/hicommonwealth/commonwealth/pull/4865/commits/94a01227f5dd0f1b6f2f968eb1d5a6d4053e2efd)